RELEASE.md

```diff
@@ -19,10 +19,10 @@ git fetch origin
 git checkout origin/master
 ```
 
-Tag the commit:
+Tag (signed) the commit:
 
 ```bash
-git tag -a vAA.BB.CC origin/master -m vAA.BB.CC
+git tag -s vAA.BB.CC origin/master -m vAA.BB.CC
```

Comment:
hmmm I don't have gpg setup, this would be a pain for me. Any reason for this?

Reply:
https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification
>  These tags or commits are marked as verified on GitHub so other people can trust that the changes come from a trusted source.

Essentially it proves that you actually made the commit or tag and that someone didn't just `GIT_COMMITER_EMAIL="user@domain" git commit --author="user@domain"` you in.

As for the significance of tag signing, I'll defer to others:
* https://security.stackexchange.com/a/196099
* https://stackoverflow.com/a/12188977

It looks like signed tags have been hit or miss in past releases, https://github.com/packethost/packet-python/tags.  I can turn a blind eye... (for now 👁️). 

The main reason I changed this to `-s` is that once `RELEASES.md` exists (and until more automation makes it irrelevant), I will copy/paste these instructions when needed. I don't want to think about them and I especially don't want to have to hand-edit them.

Here are some steps, if you are interested:
* https://docs.github.com/en/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account
* https://dev.to/wes/how2-using-gpg-on-macos-without-gpgtools-428f